File: Dev/Assets/luaScripts/config/data/GoldCarryConfig.lua
--表名: 上庄筹码, 字段描述：_key:ID, _gold_carry:上庄携带金额, 
local M = {}
M["1"] = {key = "1", gold_carry = "20000000", }
M["2"] = {key = "2", gold_carry = "30000000", }
M["3"] = {key = "3", gold_carry = "50000000", }
M["4"] = {key = "4", gold_carry = "100000000", }
M["5"] = {key = "5", gold_carry = "500000000", }
LuaConfigMgr.GoldCarryConfigLen = 5
LuaConfigMgr.GoldCarryConfig = M

File: Dev/Assets/Scripts/Config/export_cs_model/GoldCarryConfig.cs
//表名: 上庄筹码, 字段描述：_key:ID, _gold_carry:上庄携带金额,  -->
using System;
using System.Collections.Generic;
using Mono.Xml;
using System.Security;
using Utils;

public class GoldCarryConfigItem:ConfigDataItemBase{
//	private System.Object _key;
	/// <summary>
	/// ID
	/// </summary>
	public string key { get { return (string)_key; } }
	private System.Object _gold_carry;
	/// <summary>
	/// 上庄携带金额
	/// </summary>
	public int gold_carry { get { return (int)_gold_carry; } }

	public override void Parse(SecurityElement element) {
		_key = element.Attribute("key");base["key"]=_key;
		_gold_carry = Convert.ToInt32(element.Attribute("gold_carry"));base["gold_carry"]=_gold_carry;
	}
}

public class GoldCarryConfig : ConfigDataBase<GoldCarryConfigItem> {
	public GoldCarryConfig(){
		_fileName = "export_xml/gold_carry_config.bytes";
	}
}